ui-speedspacechart/src/components/helpers/drawElements/reticle.ts

```diff
@@ -181,14 +174,10 @@ export const drawCursor = ({ ctx, width, height, store }: DrawFunctionParams) =>
   previousGradientText = slopes.findLast(({ position }) => position.start <= cursorPosition)!.value;
 
   // DRAWING
-
-  // Crosshair
-  ctx.beginPath();
-  ctx.moveTo(reticleX - RETICLE_LINE, reticleY);
-  ctx.lineTo(reticleX + RETICLE_LINE, reticleY);
-  ctx.moveTo(reticleX, reticleY - RETICLE_LINE);
-  ctx.lineTo(reticleX, reticleY + RETICLE_LINE);
-  ctx.stroke();
+  ctx.font = 'normal 12px IBM Plex Sans';
+  ctx.fillStyle = 'rgb(0,0,0)';
+  ctx.shadowOffsetY = 0;
+  ctx.shadowBlur = 0;
```

Comment:
why did you add those?

Reply:
I took this code from the top of the file. I don't think they're of any use and can be deleted at the same time.